docs/website/docs/Introduction.md

```diff
@@ -17,9 +17,6 @@ This document provides an index to different Consensus documents:
   foundations.
 - [Style guide](StyleGuide.md): describes the style guide we use when developing
   Consensus.
-- [Interface changelog](interface-CHANGELOG.md): contains the old changelog,
```

Comment:
Just FYI, this file will be removed by [this pr](https://github.com/input-output-hk/ouroboros-consensus/pull/55/files)